doc/changes/devel/12366.newfeature.rst

```diff
@@ -0,0 +1 @@
+Add support for `dict` type argument ``ref_channels`` to :func:`mne._fiff.reference.set_eeg_reference`, by `Qian Chu`_.
```

Comment:
Could you explain what that allows users to do?

Reply:
Sorry that I wasn't too familiar with the private/public API and simply copied the path to the function. Does :func:`mne.set_eeg_reference` work?

Reply:
Users will rarely call this function directly though, I suppose. They will usually call the methods of Raw, Epochs, ..., objects, so I believe these should be referenced or briefly mentioned